src/schemas/components/SchemaFormProperty.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <p-label class="schema-form-property" :label="label">
+    <template v-if="description" #description>
+      <div class="schema-form-property__description">
+        <p-markdown-renderer :text="description" class="schema-form-input__markdown" />
+      </div>
+    </template>
+
+    <SchemaFormPropertyInput v-model:value="value" :property="property" />
+  </p-label>
+</template>
+
+<script lang="ts" setup>
+  import { computed } from 'vue'
+  import SchemaFormPropertyInput from '@/schemas/components/SchemaFormPropertyInput.vue'
+  import { SchemaProperty } from '@/schemas/types/schema'
+  import { SchemaValue } from '@/schemas/types/schemaValues'
+
+  const props = defineProps<{
+    property: SchemaProperty,
+    value: SchemaValue,
+    required: boolean,
+  }>()
+
+  const emit = defineEmits<{
+    'update:value': [SchemaValue],
+  }>()
+
+  const value = computed({
+    get() {
+      return props.value
+    },
+    set(value) {
+      emit('update:value', value)
+    },
+  })
+
+  const label = computed(() => {
+    const title = props.property.title ?? ''
+
+    if (!props.required) {
+      return `${title} (Optional)`.trim()
```

Comment:
This is a nitpick but I'd like to see the `(Optional)` label treated as less important than the title of the property itself, ESPECIALLY because these tend to get displayed a lot in the same contexts and they can be visually noisy. Could be as simple as making the optional text subdued or approaching it from the opposite direction and adding a chip for "Required" fields. Food for thought

Reply:
Agreed. Lets handle that in a future PR 👍